### ISSUE TITLE
Add support for missing yearn v1 vaults

### ISSUE THREAD

**Issue by LefterisJP** [rotki/rotki#2221]
## Abstract

At the moment we are missing some yearn v1 vaults that have been added since after we finished the integration. Go through their vaults (https://v1.yearn.finance/vaults) and the ones we currently support (https://github.com/rotki/rotki/blob/c2a97169a9af7df46e6bdd6d5237619a37ffec59/rotkehlchen/chain/ethereum/yearn/vaults.py#L49) and add any ones missing

## Task

Do it

**Comment by rudygt** [comment]
Hello, I have been helping a little with the yearn exporter and got familiar in how to get the v1 vaults from the registry, this may help with this issue. 

json data
https://gist.github.com/rudygt/f6e8294458bb1ac03a59cfd8d9445902

python script to get the data using brownie
https://gist.github.com/rudygt/71e1500fb35cd51b6005634b94ce6ecd

kind regards.

**Comment by LefterisJP** [comment]
This may indeed be helpful. Thank you for taking the time to link those to us @rudygt

**Comment by WyseNynja** [comment]
I think the yearn v2 vaults can probably be added to the same place. They have different code for how they handle strategies, but their price and balance checking functions are the same.

@rudygt's code probably needs a different registry address for the v2 vaults though. I'm not sure what it is.

**Comment by yabirgb** [comment]
> @rudygt's code probably needs a different registry address for the v2 vaults though. I'm not sure what it is.

The registry for v2 is v2.registry.ychad.eth . Got it from 
https://discord.com/channels/734804446353031319/735311380646723684/820358862707490847

**Comment by rudygt** [comment]
v2 vaults work a little diferent, the registry dont hold the list of vaults, we need to pull them from events, 

https://github.com/rudygt/yearn-exporter/blob/use_registry_to_get_v2_vaults/yearn/vaults_v2.py#L109

my code here should work, let me know if you need me to run it and get the json data

regards.

**Comment by rudygt** [comment]
I did went ahead and pulled the data 

https://gist.github.com/rudygt/9b42dcee825a11ae3929423837dbc6f1

and the experimental vaults, 

https://gist.github.com/rudygt/7549b5dbc308e0b9341568327f19e192

**Comment by LefterisJP** [comment]
Thanks @rudygt. This will come in handy!

**Comment by LefterisJP** [comment]
This was done via the various PRs @yabirgb made.

**Comment by yabirgb** [comment]
It was a misunderstanding. This one is not yet done. With the new logic of protocols we can remove [this list](https://github.com/rotki/rotki/blob/develop/rotkehlchen/chain/ethereum/modules/yearn/vaults.py#L75) that we keep and keep this logic in the assets queries

**Comment by LefterisJP** [comment]
This is still not done @yabirgb  eh?

**Comment by yabirgb** [comment]
@LefterisJP No, we didn't update the list of V1 vaults. Now all the active vaults are v2 but for accounting porpouses it will be needed to add all the v1

**Comment by LefterisJP** [comment]
All right so please do this at some point

**Comment by LefterisJP** [comment]
@yabirgb ... we there yet?
